frontend/src/LogsDialog.vue

```diff
@@ -24,11 +24,23 @@
 * =========================LICENSE_END==================================
 */
 <template>
-   <dlg ref="dialog" :title="`Logs of ${$route.params.service} in ${$route.params.app}`" :large="true" @close="clearLogs">
+   <dlg ref="dialog" :title="`Logs of ${$route.params.service} in ${$route.params.app}`" :large="true" @close="clearLogs"
+      :errorStatusText="this.errorText">
       <template v-slot:body>
-         <DynamicScroller :items="logLines" :item-size="20" class="ra-logs">
+         <div class="d-flex justify-content-end align-items-center">
+            <alert type="alert" class="alert-primary mr-auto" v-if="scrollPosition" role="alert">
```

Comment:
Isn't following better because more robust?

```suggestion
            <alert type="alert" class="alert-primary mr-auto" v-if="scrollPosition === 0" role="alert">
```